src/app/search-page/search-page.component.ts

```diff
@@ -42,12 +57,23 @@ export class SearchPageComponent implements OnInit, OnDestroy {
     return list.filter(filterFn);
   }
 
-  onGetCocktails (data) {
-    if (data.length) {
-      this.filteredByIngredientCocktails = data;
-    } else {
-      this.filteredByIngredientCocktails = this.cocktailsList;
+  onGetCocktails ({data, category}) {
+    const isEl = this.fullCocktailsLists.find(el => {
+      if (el.category === category) {
```

Comment:
can we move this expression to some variable? ```const isEqualCategories = el.category === category;```

Reply:
done